prolog/tokenize.pl

```diff
@@ -58,10 +59,13 @@
 %   * spaces(+bool) : Determines whether spaces are represted as tokens or discarded.
 %   * cntrl(+bool)  : Determines whether control characters are represented as tokens or discarded.
 %   * punct(+bool)  : Determines whether punctuation characters are represented as tokens or discarded.
-%   * to(+on_of([strings,atoms,chars,codes])) : Determines the representation format used for the tokens.
-%   * pack(+bool)   : Determines whether tokens are packed or repeated.
+%   * to(+one_of([strings,atoms,chars,codes])) : Determines the
```

Comment:
Thanks for correcting the typo!

Looks like the lines breaks got mangled here a bit. Should be a new line before `* pack...`

Reply:
fixed